packages/component-library/src/components/plmg-radio-button/plmg-radio-button.scss

```diff
@@ -0,0 +1,127 @@
+@use '~@ducky/plumage-tokens/dist/scss/default' as tokens;
+
+// Component style
+// -------------------------------------------------
+.radio-button-wrapper {
+  display: flex;
+  cursor: pointer;
+  align-items: center;
+  width: fit-content;
+  border-radius: tokens.$plmg-border-radius-s;
+
+  &:hover {
+    input[type='radio'] {
+      background-color: tokens.$plmg-color-background-primary-strong;
+      border-color: tokens.$plmg-color-border-primary-strong;
+      transition: background-color 0.2s ease, border-color 0.2s ease;
+
+      &:checked {
+        background-color: tokens.$plmg-color-background-neutral-hover;
+        border-color: tokens.$plmg-color-border-primary-strong;
+        & ~ label {
+          color: tokens.$plmg-color-text-neutral;
+        }
+      }
+    }
+
+    input[type='radio']:checked::after {
+      box-shadow: inset 1em 1em
+        tokens.$plmg-color-background-primary-strong-hover;
+    }
+
+    label {
+      color: tokens.$plmg-color-text-primary-strong;
+    }
+  }
+
+  &:active {
+    input[type='radio']:checked::after {
+      box-shadow: inset 1em 1em
+        tokens.$plmg-color-background-primary-strong-active;
+    }
+  }
+
+  &:focus-within {
+    outline: tokens.$plmg-border-width-l solid
+      tokens.$plmg-color-border-standout;
+  }
+}
+
+input[type='radio'] {
+  -webkit-appearance: none;
+  appearance: none;
+  margin: 0;
+  background-color: tokens.$plmg-color-background-neutral;
+  font: inherit;
+  width: 20px;
+  height: 20px;
+  border: tokens.$plmg-border-width-s solid tokens.$plmg-color-border-neutral;
+  border-radius: tokens.$plmg-border-radius-circle;
+  box-shadow: tokens.$plmg-shadow-inner-s;
+  margin-right: tokens.$plmg-spacing-x0-5;
+  cursor: pointer;
+  position: relative;
+
+  &.highlighted {
+    border: tokens.$plmg-border-width-s solid
+      tokens.$plmg-color-border-danger-strong;
+  }
+
+  &:checked {
+    box-shadow: none;
+    border: tokens.$plmg-border-width-s solid
+      tokens.$plmg-color-border-primary-strong;
+  }
+
+  &.large {
+    width: 22px;
+    height: 22px;
+    margin-right: tokens.$plmg-spacing-x0-75;
+    &::after {
+      width: 14px;
+      height: 14px;
+    }
+  }
+
+  &:focus-visible {
+    outline: none;
+  }
+}
+
+input[type='radio']::after {
+  content: '';
+  width: 12px;
+  height: 12px;
+  border-radius: tokens.$plmg-border-radius-circle;
+  transform: scale(0);
+  transition: 50ms transform ease-in-out;
+  box-shadow: inset 1em 1em tokens.$plmg-color-background-primary-strong;
+  position: absolute;
+  top: 3px;
+  left: 3px;
+}
+
+input[type='radio']:checked::after {
+  transform: scale(1);
+}
+
+label {
```

Comment:
Same here, you must use a custom class or encapsulate it within `.radio-button-wrapper` or `.plmg-radio-button`.
Or use a CSS selector: `.plmg-radio-button + label` would work (I think)